lib/OnyxUtils.ts

```diff
@@ -311,17 +311,14 @@ function tryGetCachedValue<TKey extends OnyxKey>(key: TKey, mapping?: Partial<Wi
             return;
         }
 
-        const matchingKeys = Array.from(allCacheKeys).filter((k) => k.startsWith(key));
-        const values = matchingKeys.reduce((finalObject: NonNullable<OnyxCollection<KeyValueMapping[TKey]>>, matchedKey) => {
-            const cachedValue = cache.getValue(matchedKey);
-            if (cachedValue) {
-                // This is permissible because we're in the process of constructing the final object in a reduce function.
-                // eslint-disable-next-line no-param-reassign
-                finalObject[matchedKey] = cachedValue;
+        const values: Record<string, unknown> = {};
```

Comment:
```suggestion
        const values: OnyxCollection<KeyValueMapping[TKey]> = {};
```

Reply:
Hello @fabioh8010!  
I am going to take over the PR from Hur, cause he is on the leave. I applied suggested changes, please have an another look :)